app/src/main/java/com/onboarding/parkingsystemkotlin/utils/ConstantUtils.kt

```diff
@@ -4,4 +4,5 @@ object ConstantUtils {
     const val PARKING_VIEW_TAG: String = "ParkingView"
     const val CONFIGURE_PARKING_LOTS_DIALOG_TAG = "ConfigureParkingDialog"
     const val PARKING_LOT_NOT_SET: Int = -1
+    const val RESERVATION_ACTIVITY_TAG: String = "ReservationActivity";
```

Comment:
please remove de ; as it is not necessary

Reply:
removed